baseProject/App.tsx

```diff
@@ -12,6 +12,14 @@ import { BottomTabNavigator } from './src/common/navigation/bottomTab/BottomTabN
 {{else}}
 import { HomeContainer } from './src/features/home/container/HomeContainer'
 {{/if}}
+{{#if hasGluestackUIDefault}}
+import { GluestackUIProvider } from '@gluestack-ui/themed';
+import {config } from '@gluestack-ui/config';
+{{/if}}
+{{#if hasGluestackUIThemedMM}}
+import { GluestackUIProvider } from '@gluestack-ui/themed';
+import { config } from './config/gluestack-ui-MM.config';
+{{/if}}
```

Comment:
@hristo87 Could you please try this `handlebars` expression? It should work since we use handlebars v4+.
```suggestion
{{#if hasGluestackUI}}
  import { GluestackUIProvider } from '@gluestack-ui/themed';
  {{#if hasGluestackUIDefault}}
  import {config } from '@gluestack-ui/config';
  {{else if hasGluestackUIThemedMM}}
  import { config } from './config/gluestack-ui-MM.config';
  {{/if}}
{{/if}}
```

Reply:
Done / Tested